### PR TITLE
build: switch user to appsody-devops

### PR DESCRIPTION
**Pull request by tomleah**
- Switching the GitHub tokens 

Once this is in we should change the token in Travis settings

### CODE REVIEW

**Comment by tomleah** [comment]
Rebased 🙂

File: .travis.yml [hunk 1]
```diff
@@ -57,8 +57,7 @@ jobs:
       deploy:
         provider: releases
         skip_cleanup: true
-        api_key:
-          secure: gtGyAu3m2v26KtkzQu4ljkLGLGKpCJAak0cO00zlvDvEGxwqmiH7yytG8uoAOise8Fp/1pF5YXrhpw6I7+WN1s8k+GJyCKoLG60wf9ODF4BR+9vGiwOfObs91yAfKz4oA/zzGG2eLdD59Y4VoRgY46OzsmGDbw9Zf34r+xGQ3fdrQCYbLItJzLwX/GEfWGBtVmJVb+BIuFrZ0UF1zTswC8+TkPDj/u+Pnc5bMXUqH7gUz2wrVdDc873EJjILB6U8iMIrFrKshr1CQ/C4JTJh7l6xiCJ5NS8Ik40jYrVfytMHo9cC93VkUnl1fNJuBo7kRQAbek1Jp7gmsfX1BQzmXp6p+VpgQmealqDWa+b25VhBS7T0Bz3nZvktAdRE6kbpVYgbS/YpDEQWNLaMreezd9taYjDFz+k3PlNCsMqPGfyU4jV1VViHuyFJvBuIt3JlHOm3mF/bNIdfDLVtkugl/etBoyITD0Gths+xNh997ctB9diGhlwmu3VkMEo173aEtYjpNPLQrkAFtA0476xLIJroPb4R2KlO31iC5tpUU9Ge/65ZUPaMaDIgR8kcLGmKvAdNwdDonZguT8Q/XlEPpn0MCmKbZcX2/G3l3+Sc2YN7EHeH1t7gGbQAjzHzsQCHLCCbL2p8X/AJzf6vQJRI1kEEs9QFwLg7vBjjT2n6Nzg=
+        api_key: $GH_TOKEN
```
Comment (kylegc) [inline .travis.yml]: I recall having issues reading an env variable here, but maybe that was in an older travis enterprise version. Unfortunately it is hard to test this without actually submitting a release.

File: deploy-build/deploy.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ cd homebrew-appsody
 git checkout -b test${TRAVIS_TAG}${TRAVIS_BUILD_NUMBER}
 cp ../../package/appsody.rb .
 git add appsody.rb
-git commit -m "Travis build: $TRAVIS_TAG$TRAVIS_BUILD_NUMBER" --author="Kyle G. Christianson <christik@us.ibm.com>"
+git commit -m "Travis build: $TRAVIS_TAG$TRAVIS_BUILD_NUMBER" --author="Appsody Devops <appsodydev@gmail.com>"
```
Comment (kylegc) [inline deploy-build/deploy.sh]: This "user" will need to sign the Appsody CLA. Is there a github account associated with this email?

File: .travis.yml (hunk 1 above)
Comment (kylegc) [reply .travis.yml]: Looks like the stacks repo is using a variable here, so I'm willing to give it a go!

Comment (tomleah) [reply .travis.yml]: Yeah the stacks and the website repo use env variables so i think its okay

File: deploy-build/deploy.sh (hunk 2 above)
Comment (tomleah) [reply deploy-build/deploy.sh]: Yes good catch! @neeraj-laad should have access to this account so sign the CLA

**Pull request merged**